deploy/internal-registry/eclipse/cloud-shell/nightly/meta.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v2
```

Comment:
I would say that it's not a part of `deploy` configuration but I'm not sure.
I can propose to make `internal-registry` as a root folder or subfolder of `build` folder...